nixos/modules/services/misc/matrix-synapse.nix

```diff
@@ -44,7 +44,7 @@ database: {
 }
 event_cache_size: "${cfg.event_cache_size}"
 verbose: ${cfg.verbose}
-log_file: "/var/log/matrix-synapse/homeserver.log"
+log_file: "${cfg.dataDir}/homeserver.log"
```

Comment:
Do we even want a separate logfile? All logs are going to the systemd jorunal anyways.

Reply:
@fpletz I don't have strong feelings either way - should I amend this PR or make another one to remove it?

Reply:
I think two separate commits for to unrealted changes should do, however I don't think that it's necessary to even open another PR :)

Reply:
@fpletz @Ma27 alright, I've added another commit that removes that line, and it seems to not have had any negative effect on my homeserver 👍